cli/install/cni-template.go

```diff
@@ -0,0 +1,194 @@
+package install
+
+// CNITemplate provides the base template for the `linkerd install-cni-plugin` command.
+const CNITemplate = `# Copyright 2017 CNI authors
+# Modifications copyright (c) Nordstrom, Inc
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file was inspired by:
+# 1) https://github.com/istio/cni/blob/c63a509539b5ed165a6617548c31b686f13c2133/deployments/kubernetes/install/manifests/istio-cni.yaml
```

Comment:
I appreciate you adding the licenses throughout this diff, but in this particular case my vote is to omit it. It's a bit confusing for the CLI to emit a license, and this is mostly Kubernetes scaffolding anyway. If you want to add attribution, you could add it at the top of this file, not as part of the `CNITemplate` string.